### PR TITLE
more mappings for SpawnHelper and ocean monument

### PR DESCRIPTION
**Pull request by valoeghese**

### CODE REVIEW

**Review by Runemoro — APPROVED** [review]

**Comment by valoeghese** [comment]
review pls

File: mappings/net/minecraft/structure/OceanMonumentGenerator.mapping [hunk 1]
```diff
@@ -8,13 +8,21 @@ CLASS net/minecraft/class_3366 net/minecraft/structure/OceanMonumentGenerator
...
 	CLASS class_3383 Penthouse
-	CLASS class_3384
+	CLASS class_3384 Piece
 		FIELD field_14471 SEA_LANTERN Lnet/minecraft/class_2680;
 		FIELD field_14472 ICE_BLOCKS Ljava/util/Set;
 		FIELD field_14473 PRISMARINE Lnet/minecraft/class_2680;
 		FIELD field_14474 DARK_PRISMARINE Lnet/minecraft/class_2680;
 		FIELD field_14475 WATER Lnet/minecraft/class_2680;
 		FIELD field_14476 PRISMARINE_BRICKS Lnet/minecraft/class_2680;
+		METHOD method_14773 (Lnet/minecraft/class_1936;Lnet/minecraft/class_3341;IIIIII)V
```
Comment (liach) [inline mappings/net/minecraft/structure/OceanMonumentGenerator.mapping]: What's the method's name

Comment (valoeghese) [reply mappings/net/minecraft/structure/OceanMonumentGenerator.mapping]: setAirAndWater

**Review by liach — APPROVED** [review]

**Review by Runemoro — CHANGES_REQUESTED** [review]
Ignore this, wrong PR.

**Pull request merged**